### PR TITLE
feat: adds cancelled trips

### PR DESCRIPTION
**Pull request by mikaelbr**

### CODE REVIEW

**Comment by mortennordseth** [comment]
Verified that this includes cancelled trips: 

image

**Comment by mortennordseth** [comment]
Screenshots of the implementation: 

image

image

**Comment by mortennordseth** [comment]
Any suggestions for how to improve the mobile view is appreciated. Could maybe remove the left icon for smaller devices? 🤔 

image

**Comment by toretefre** [comment]
Isn't a greyed out tile background colour used other places to make it even more explicit that something is cancelled?

**Comment by mortennordseth** [comment]
> Isn't a greyed out tile background colour used other places to make it even more explicit that something is cancelled?

It's used for trips that have passed. That may be confusing. The warning icon makes it different though. The departure page shows canceled departures like this: 

image

The problem with strike-through is that it can be hard to read the text, so I went with the text (Kansellert) behind the header.

File: src/page-modules/assistant/details/details-header/index.tsx [hunk 1]
```diff
@@ -36,27 +36,57 @@ export default function DetailsHeader({ tripPattern }: DetailsHeaderProps) {
...
+              PageText.Assistant.trip.tripPattern.isCancelled.title,
+            ).toUpperCase()})`}
+        </Typo.h2>
+      </div>
       <div className={style.tripDetails}>
         <div className={style.date}>
           <MonoIcon icon="time/Date" />
-          <Typo.p textType="body__primary">{weekdayAndDate}</Typo.p>
+          <Typo.p
+            textType={isCancelled ? 'body__primary--strike' : 'body__primary'}
```
Comment (adriansberg) [inline src/page-modules/assistant/details/details-header/index.tsx:73]: IIRC, in other places where we use strike-through to indicate that it is cancelled, the colour of the text is also changed. Should it be here as well? (I don't know how the contrast ratio is with a colour change.)

**Review by adriansberg — APPROVED** [review]
LGTM

File: src/page-modules/assistant/details/details-header/index.tsx (hunk 1 above)
Comment (mortennordseth) [reply src/page-modules/assistant/details/details-header/index.tsx:73]: You are correct. The secondary text color is used on the departure page to indicate a canceled departure. I talked to the product owner and he agreed that we could iterate on this in the next release. We should also talk to the designers at AtB do look into this when implementing it in the app

**Pull request merged**